blueprint-skeleton/README.md.tmpl

```diff
@@ -0,0 +1,64 @@
+# _BLUEPRINT_NAME_
```

Comment:
can be `{{.BlueprintName | uppercase}}`

Reply:
Using `{{.BlueprintName | replace "-" " " | title}}`

Reply:
This turns `my-amazing-blueprint` into `My Amazing Blueprint`